Samples/SparsePackages/SparseCppSample/AppxManifest.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Package IgnorableNamespaces="uap uap10"
+  xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10"
+  xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10"
+  xmlns:uap10="http://schemas.microsoft.com/appx/manifest/uap/windows10/10"
+  xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities">
+  <Identity Name="SparseCppSample" Publisher="CN=Contoso" Version="1.0.0.0" ProcessorArchitecture="neutral" />
+  <Properties>
+    <DisplayName>Sparse Cpp Sample</DisplayName>
```

Comment:
"Package with External Location C++ Sample"

Reply:
Generally remove "sparse package" references throughout in favor of "Package with External Location" to align with the public documentation.

Reply:
FYI I'll be fixing up the C# sample.